src/shared/components/oncoprint/ResultsViewOncoprint.tsx

```diff
@@ -173,8 +173,24 @@ export default class ResultsViewOncoprint extends React.Component<
         );
     }
 
-    @computed get selectedTreatmentsFromUrl() {
-        return this.props.store.treatmentList;
+    @computed get selectedGenericAssayEntitiesGroupByGenericAssayTypeFromUrl() {
+        const result = _.reduce(
+            this.props.store.selectedGenericAssayEntities,
+            (acc, value, key) => {
+                if (
+                    this.props.store.molecularProfileIdToMolecularProfile
+                        .result[key]
+                ) {
+                    const type = this.props.store
+                        .molecularProfileIdToMolecularProfile.result[key]
```

Comment:
note that we must make sure status of this result is complete.   just make sure that this is not accessed without checking for status